test/components.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import _ from 'lodash';
+import faker from 'faker';
+import { join } from 'path';
+import { create } from 'react-test-renderer';
+import generateListing from '../util/generateListing';
+import Description from '../client/src/components/Description';
+import 'babel-polyfill'; // this is needed to make async/await work
+import 'jest-styled-components';
+
+describe('component tests', () => {
+  test('it should render a description component', async () => {
+    jest.setTimeout(90000);
+    // make sure we always generate the same listing object
+    jest.unmock('lodash');
+    _.random = (min) => min;
+    _.sample = (collection) => collection[0];
+    faker.seed(1337);
+    const imagesPath = join(__dirname, 'testImages');
+    const listing = await generateListing('001', '001', imagesPath);
```

Comment:
I would suggest looking into using hard coded mock data instead of generating a new listing all together. Snapshots would have to be overwritten every time the test is run since this function has some randomness to it. Calling generateListing is also more expensive that writing mock data